### DISCUSSION TITLE
support for sample_weights in some of the numerical transformers

### DISCUSSION THREAD

**Discussion by duffar12** [feature-engine/feature_engine#682] [Ideas]
Hi,

i just discovered your package and it looks great! I was just wondering if you might have thought about adding support for sample_weights in some of the numerical transformers? 
For example in the box-cox/yeo-johnson transformers we're trying to get the data into a gaussian shape. However sample weighting can  have an impact on the output of this

**Comment by solegalli** [comment]
Hey @duffar12 

Yes, I'd love to add support for sample_weights. It is a lot of work though. So I will need some help. I welcome contributions from all walks of life, so if you fancy making this implementation for a particular transformer, please go ahead.

Slowly but steadily, we are getting there ;)

Thanks for the suggestion!
